src/NuGet.Clients/NuGet.PackageManagement.UI/Xamls/InfiniteScrollList.xaml

```diff
@@ -311,7 +314,17 @@
         DismissClick="_loadingStatusBar_DismissClick"
         Style="{StaticResource FadeAnimationStyle}"
         Visibility="Hidden"/>
+
+      <!--
+        Special TextBlock to report loading status to assistive technologies (narrator).
+        Code behind changes in Text property trigger a narrator event
+      -->
+      <vs:LiveTextBlock
```

Comment:
This causes narrator to read the control content when using Caps Lock + Arrow key, even if it is collapsed

Reply:
Since we want to report status when it is loading, I will remove the report from other status changes, as it makes difficult to achieve a proper status change notification to the narrator.

Reply:
Now that this goes to P4, I will try to have a more complete solution.